docs/proposals/kubelet-eviction.md

```diff
@@ -69,6 +69,7 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | Eviction Signal  | Description                                                                     |
 |------------------|---------------------------------------------------------------------------------|
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
+| disk.available   | disk.available := node.stats.fs.available |
```

Comment:
On another hand, using percentage to describe this threshold has caveat too. For example, the threshold is 1% of disk size, on a node with 100G means 1G, while another node with 200G means 2G, which leads resource utilization issues.